Makefile.toml

```diff
@@ -115,10 +115,7 @@ description = "Runs prettier to check appropriate JS code format."
 workspace = false
```

Comment:
:upside_down_face: I can't comment on the line, because it's too far away from the changes, but I think this description should be different to the one below since this command actually modifies the files, i.e. "Runs prettier to format JS code appropriately".